packages/aom.cmake

```
@@ -5,7 +5,7 @@ ExternalProject_Add(aom
     GIT_REMOTE_NAME origin
     GIT_TAG main
     UPDATE_COMMAND ""
-    CONFIGURE_COMMAND ${EXEC} cmake -H<SOURCE_DIR> -B<BINARY_DIR>
+    CONFIGURE_COMMAND ${EXEC_CONF} cmake -H<SOURCE_DIR> -B<BINARY_DIR>
```

```suggestion
    CONFIGURE_COMMAND CONF=1 ${EXEC} cmake -H<SOURCE_DIR> -B<BINARY_DIR>
```
I would prefer adding `CONF=1` variable on every configure command. Instead of duplicating exec file

---

done